Marlin/Marlin_main.cpp

```diff
@@ -5018,47 +5018,32 @@ inline void gcode_G28() {
       #endif
 
       const int8_t pp = code_seen('C') ? code_value_int() : DELTA_CALIBRATION_DEFAULT_POINTS,
-                   probe_points = (WITHIN(pp, 1, 4) || pp == -2) ? pp : DELTA_CALIBRATION_DEFAULT_POINTS;
+                   probe_points = (WITHIN(pp, -7, -1) || WITHIN(pp, 1, 7)) ? pp : DELTA_CALIBRATION_DEFAULT_POINTS;
 
       int8_t verbose_level = code_seen('V') ? code_value_byte() : 1;
 
-      #if ENABLED(DELTA_CALIBRATE_EXPERT_MODE)
-        #define _MAX_M33_V 3
-        if (verbose_level == 3 && probe_points == 1) verbose_level--; // needs at least 4 points
-      #else
-        #define _MAX_M33_V 2
-        if (verbose_level > 2)
-          SERIAL_PROTOCOLLNPGM("Enable DELTA_CALIBRATE_EXPERT_MODE in Configuration_adv.h");
-      #endif
-
-      if (!WITHIN(verbose_level, 0, _MAX_M33_V)) verbose_level = 1;
+      if (!WITHIN(verbose_level, 0, 2)) verbose_level = 1;
 
-      float zero_std_dev = verbose_level ? 999.0 : 0.0; // 0.0 in dry-run mode : forced end
+      float zero_std_dev = verbose_level ? 999.0 : 0.0;                                          // 0.0 in dry-run mode : forced end
 
       gcode_G28();
 
       float e_old[XYZ],
             dr_old = delta_radius,
-            zh_old = home_offset[Z_AXIS];
+            zh_old = home_offset[Z_AXIS],
+            alpha_old = delta_tower_angle_trim[A_AXIS],
+            beta_old = delta_tower_angle_trim[B_AXIS];
       COPY(e_old,endstop_adj);
-      #if ENABLED(DELTA_CALIBRATE_EXPERT_MODE)
-        // expert variables
-        float h_f_old = 1.00, r_f_old = 0.00,
-              h_diff_min = 1.00, r_diff_max = 0.10;
-      #endif
 
       // print settings
 
       SERIAL_PROTOCOLLNPGM("G33 Auto Calibrate");
       SERIAL_PROTOCOLPGM("Checking... AC");
       if (verbose_level == 0) SERIAL_PROTOCOLPGM(" (DRY-RUN)");
-      #if ENABLED(DELTA_CALIBRATE_EXPERT_MODE)
-        if (verbose_level == 3) SERIAL_PROTOCOLPGM(" (EXPERT)");
-      #endif
       SERIAL_EOL;
       LCD_MESSAGEPGM("Checking... AC");
 
-      SERIAL_PROTOCOLPAIR("Height:", DELTA_HEIGHT + home_offset[Z_AXIS]);
+      SERIAL_PROTOCOLPAIR(".Height:", DELTA_HEIGHT + home_offset[Z_AXIS]);
```

Comment:
Indeed everything from Marlin is wrongly aligned in Repetier- Host as well as in Pronterface  (eg G29 output the column numbers do not correspond to the table below) because of those leading spaces being suppressed (very enjoying feature of Marlin not taking this into account). The output from the iteration routine is already difficult to get, so if everything is all over the place it is pure mayhem. And I do not think I'm the only one using those apps.

Hopefully this will set a good precedent and ALL the Marlin output would become more readable.